site/cds_rdm/migration/data/transform/.gitkeep

```diff
@@ -0,0 +1 @@
+# This file exists only to add templates/ folder to git repository.
```

Comment:
review strings? Why do we need this folder?

Reply:
because I have hardcoded it in the streams.yaml, less things to be remembered to modify

Reply:
ah sorry, I meant that the string is wrong
```suggestion
# This file exists only to add cds_rdm/migration/data/transform/ folder to git repository.
```